app/helpers/application_helper.rb

```diff
@@ -648,5 +648,7 @@ def non_owner_participation_rule(employer)
     end
   end
 
-
+  def individual_market_is_enabled?
+    Settings.aca.market_kinds.include?("individual")
+  end
```

Comment:
This is fine for now - I know you pulled from Lokesh's branch. I have them in individual concerns / helpers on my branch but this is a minor change I'll take care of when I merge the various PRs.